### PR TITLE
refactor: adding config stem constants

### PR DESCRIPTION
**Pull request by ItsRico**
refactor: Adding Config Stem Constants 

Added constants to be common that are previously defined only in edgex-go.

Closes: #754

If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Review by bnevis-i — DISMISSED** [review]

**Comment by bnevis-i** [comment]
@ItsRico   DCO check is failing.  Need to sign the commit.

File: common/constants.go [hunk 1]
```diff
@@ -303,3 +303,9 @@ const (
 	DeviceSystemEventActionUpdate = "update"
 	DeviceSystemEventActionDelete = "delete"
 )
+
+const (
+	ConfigStemApp    = "edgex/appservices/"
+	ConfigStemCore   = "edgex/core/"
+	ConfigStemDevice = "edgex/devices/"
+)
```
Comment (lenny-intel) [inline common/constants.go:311]: Missing `ConfigStemSecurity = "edgex/security/"`

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: common/constants.go (hunk 1 above)
Comment (bnevis-i) [reply common/constants.go:311]: I'm not sure its needed.  I checked the original PR that we are trying to refactor and `ConfigStemSecurity ` was unused.

Comment (lenny-intel) [reply common/constants.go:311]: It was missed on the original PR and all need to be moved for consistency
https://github.com/edgexfoundry/edgex-go/blob/main/internal/constants.go#L22-L25

Comment (bnevis-i) [reply common/constants.go:311]: Agreed.  Checked latest code and this is true.

**Comment by ItsRico** [comment]
Added missed constant & signed off commits

**Review by bnevis-i — APPROVED** [review]

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**